gradle-plugins/src/main/resources/docker/Dockerfile-java

```diff
@@ -19,6 +19,13 @@ ARG JAR_FILE
 ENV JAR_FILENAME=${JAR_FILE}
 COPY ${JAR_FILE} fat.jar
 
+# add public datadog certificate to truststore
+RUN openssl s_client -showcerts -connect www.ddog-gov.com:443 2>&1 |\
+    sed -ne '/-BEGIN CERTIFICATE-/,/-END CERTIFICATE-/p' > temp.pem
+RUN openssl x509 -in temp.pem -out real.pem
+RUN keytool -importcert -alias ddog -file real.pem -cacerts -storepass changeit --noprompt
```

Comment:
Maybe for follow up work but just washed to understand if this cert is something that should be managed by k8s cert manager that is described in LHDI docs